examples/keyword-extraction-api.ipynb

```diff
@@ -0,0 +1,615 @@
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e-ujphILqepu",
+   "metadata": {
+    "id": "e-ujphILqepu"
+   },
+   "source": [
+    "### Check job progress"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "iFrDrriQqizC",
+   "metadata": {
+    "id": "iFrDrriQqizC"
+   },
+   "source": [
+    "All requests are now being processed! In the following section, we’ll monitor the status of each job to see how they’re progressing. Let’s take a look and keep track of their completion."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "SuH0CfoqjP3d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Job completed!'"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "def parse_json_objects(data_string):\n",
+    "    if isinstance(data_string, bytes):\n",
+    "        data_string = data_string.decode('utf-8')\n",
+    "\n",
+    "    json_strings = data_string.strip().split('\\n')\n",
+    "    json_objects = []\n",
+    "\n",
+    "    for json_str in json_strings:\n",
+    "        try:\n",
+    "            json_obj = json.loads(json_str)\n",
+    "            json_objects.append(json_obj)\n",
+    "        except json.JSONDecodeError as e:\n",
+    "            print(f\"Error parsing JSON: {e}\")\n",
+    "\n",
+    "    return json_objects\n",
+    "\n",
+    "all_completed = False\n",
+    "while not all_completed:\n",
+    "    all_completed = True\n",
+    "    output_lines = []\n",
+    "\n",
+    "    updated_job = client.batches.retrieve(inference_job.id)\n",
+    "\n",
+    "    if updated_job.status != \"completed\":\n",
+    "        all_completed = False\n",
+    "        completed = updated_job.request_counts.completed\n",
+    "        total = updated_job.request_counts.total\n",
+    "        output_lines.append(f\"Job status: {updated_job.status} - Progress: {completed}/{total}\")\n",
+    "    else:\n",
+    "        output_lines.append(f\"Job completed!\")\n",
+    "\n",
+    "    # Clear the output and display updated status\n",
+    "    clear_output(wait=True)\n",
+    "    for line in output_lines:\n",
+    "        display(line)\n",
+    "\n",
+    "    if not all_completed:\n",
+    "        time.sleep(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "TkkhIG9HU0D9",
+   "metadata": {
+    "id": "TkkhIG9HU0D9"
+   },
+   "source": [
+    "## Get the results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "12c1f6ac-8d60-4158-9036-de79fa274983",
+   "metadata": {},
+   "source": [
+    "Now that the job is complete, we’ll fetch the results and examine the responses generated for each request."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "806a5eb1-f6d3-491d-b051-9d44bf046a7e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{'id': '67575ea4d7432ed97b937dc1',\n",
```

Comment:
As discussed, let's simplify the output so it's a JSON object that the user could process programmatically. This is just an example to demonstrate what I mean.
{
  "document_id": "unique_id",
  "original_text": "the input text",
  "keywords": [
    {
      "keyword": "extracted keyword",
      "relevance_score": 0.00,
      "frequency": 0
    }
}